lisa/tools/kdump.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.tools.service import Systemctl
+from lisa.tools.echo import Echo
+
+from pathlib import PurePosixPath
+from lisa.executable import Tool
+from lisa.tools import Sed, Cat
+from lisa.operating_system import Posix, Redhat, Suse, Debian
+from typing import cast
+from assertpy import assert_that
+
+
+class Kdump(Tool):
+    # Redhat: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/kernel_administration_guide/kernel_crash_dump_guide
+    # Ubuntu: https://ubuntu.com/server/docs/kernel-crash-dump
+    # Suse: https://www.suse.com/zh-cn/support/kb/doc/?id=000016171
+    kexec_crash = "/sys/kernel/kexec_crash_loaded"
+
+    @property
+    def command(self) -> str:
+        return "makedumpfile"
```

Comment:
This is not consistent with this tool. It may need two tools, Kdump to manage the service, and makedumpfile to create dump.

Reply:
Done. Changed into that kdump depends on kexec and makedumpfile.

Reply:
Done